.github/workflows/pytorch-version-tests.yml

```diff
@@ -62,9 +62,15 @@ jobs:
         run: |
           pytest tests -vvv
 
+  create-issue:
+    runs-on: ubuntu-latest
+    # https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#needs-context
+    needs: build
+    if: always() && needs.build.result != 'success'
```

Comment:
Can't we use `failure()` instead of `always() && ...` : https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#failure ?

Reply:
I have tried in the first commit and `create-issue` job doesn't run with `always()`

Reply:
Why can't we do : `if: needs.build.result == 'failure'` ?

Reply:
it would miss `cancelled` and `skipped` result.

Reply:
I think it is OK to skip `cancelled` as it is maintainer's action. Good point about "skipped" but I'm not sure about in which case it can appear and we'd like to be notified...

Reply:
So `== 'failure'` or `!= 'success'` ?

Reply:
`== 'failure'` is the winner :)